packages/plugin-ext/src/plugin/type-converters.spec.ts

```diff
@@ -278,7 +291,7 @@ describe('Type converters:', () => {
             }
         };
 
-        it('should convert to task dto', () => {
+        it('shell task with string command should convert to task dto', () => {
```

Comment:
minor: generally we follow titles like `should...` so we can update to:

```suggestion
        it('should convert to task dto with with string command', () => {
```